### PR TITLE
Update usa_2005.json

### PR DESCRIPTION
**Pull request by ColonelAkirNakesh**
Adjust spacing in some flyable aircraft (tabs vs spaces), 
Adds MRAP and new Abrams model
Removes Hornet livery enforcement

### CODE REVIEW

File: resources/factions/usa_2005.json [hunk 1]
```diff
@@ -114,25 +116,5 @@
...
-      "VFA-97",
-      "VMFA-122",
-      "VMFA-132",
-      "VMFA-251",
-      "VMFA-312",
-      "VMFA-314",
-      "VMFA-323"
-    ]
-  }
-}
```
Comment (Raffson) [inline resources/factions/usa_2005.json]: Why did you remove the livery overrides exactly? 😅

**Review by Raffson — CHANGES_REQUESTED** [review]

**Comment by ColonelAkirNakesh** [comment]
I thought it was redundant with the squadron system and they weren't
necessary since we have US Hornet squadrons?

Is it possible to revert those changes and still add the Abrams and MRAP or
should I delete this one and start a new pull request?

**Comment by Raffson** [comment]
Livery-overrides are still used, but as a fallback in case no livery or livery-set was defined for a specific squadron, so it's still applicable.

You can indeed revert the change on this branch, no need for a new PR 😉

**Comment by ColonelAkirNakesh** [comment]
OK... I edited and hopefully I was able to restore the liveries! Sorry for
being such a noob.

**Comment by Raffson** [comment]
No worries, you're actually doing pretty good 😉

**Review by Raffson — APPROVED** [review]

**Pull request merged**